src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/Interop/msquic_generated_linux.cs

```diff
@@ -0,0 +1,110 @@
+//
```

Comment:
@nibanks Is it possible for us to change the copyright here to:
```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
```
???

This is what the compiler/analyzer wants from me. 

Of course, I can figure out where the message comes from and whether we can skip it here in case it's not possible to change due to legal reasons.